source/change-streams/tests/unified/change-streams-showExpandedEvents.yml

```diff
@@ -0,0 +1,296 @@
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: dropIndex
+        object: *collection0
+        arguments:
+          name: *index1
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: dropIndexes
+
+  - description: "when showExpandedEvents is true, create events are reported"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: &collection1 foo
+      - name: createChangeStream
+        object: *database0
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *collection1
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: create
+
+  - description: "when showExpandedEvents is true, create events on views are reported"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: &collection1 foo
+      - name: createChangeStream
+        object: *database0
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *collection1
+          viewOn: testName
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: create
+
+  - description: "when showExpandedEvents is true, modify events are reported"
+    operations:
+      - name: createIndex
+        object: *collection0
+        arguments:
+          keys:
+            x: 1
+          name: &index2 x_2
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: runCommand
+        object: *database0
+        arguments:
+          command:
+            collMod: *collection0
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          operationType: modify
+
+  - description: "when showExpandedEvents is true, shardCollection events are reported"
+    runOnRequirements:
+      - topologies: [ sharded-replicaset, sharded ]
+    operations:
+      - name: dropCollection
+        object: *shardedDb
+        arguments:
+          collection: *shardedCollection
+      - name: createCollection
+        object: *shardedDb
+        arguments:
+          collection: *shardedCollection
+      - name: createChangeStream
+        object: *shardedCollection
+        arguments:
+          pipeline: []
+          showExpandedEvents: true
+          showSystemEvents: true
```

Comment:
Since `showSystemEvents` isn't mentioned in the spec yet, maybe this test should wait for a follow-up?